internal/release/dynatrace-otel-collector.service

```diff
@@ -6,6 +6,7 @@ Documentation=https://docs.dynatrace.com/docs/shortlink/otel-collector
 [Service]
 EnvironmentFile=/etc/dynatrace-otel-collector/dynatrace-otel-collector.conf
 ExecStart=/usr/bin/dynatrace-otel-collector $OTELCOL_OPTIONS
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
Did this come from upstream?

Reply:
yes https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/otelcol-contrib.service

Reply:
ah config reloading